services/apis/sheepdog/sheepdogProps.js

```diff
@@ -60,13 +60,16 @@ module.exports = {
   }),
 
   /**
-   * Gen3Response when no authentication provided
+   * Expected contents of the response when no authentication is provided
    */
-  resNoAuth: new Gen3Response({
-    request: {},
+  resNoAuth: {
     status: 401,
-    body: { message: 'Authentication Error: Signature has expired' },
-  }),
+    // Before sheepdog's arborist update, the error is:
+    // "Authentication Error: Signature has expired"
+    // After sheepdog's arborist update, the error is:
+    // "request to arborist failed: error decoding token: expired at time: 1567720295"
```

Comment:
Oh also fwiw this is a real timestamp 😆

Reply:
yeah i thought it was fine but i can change it :p